src/riak_kv_index_hashtree.erl

```diff
@@ -370,7 +403,8 @@ handle_cast(build_failed, State) ->
     {noreply, State2};
 handle_cast(build_finished, State) ->
     State2 = do_build_finished(State),
-    {noreply, State2};
+    State3 = do_inform_waiting(State2),
```

Comment:
We cast a message if the tree gets built, but remain silent if the build failed (function clause above this).